pandas/plotting/_matplotlib/core.py

```diff
@@ -54,6 +54,7 @@
 )
 from pandas.core.dtypes.missing import isna
 
+from pandas import Series
```

Comment:
Can we conditionally import this where used?

Reply:
Done